src/library/Button/Button.js

```diff
@@ -163,7 +172,8 @@ const styles = {
             return theme.Button_backgroundColor_focus;
           }
         })(),
-        boxShadow: theme.Button_boxShadow_focus,
+        borderColor: minimal ? theme.Button_borderColor_focus : undefined,
+        boxShadow: minimal ? null : theme.Button_boxShadow_focus,
```

Comment:
Is there a difference in using `null` vs. `undefined` on these 2 lines?

Reply:
Probably not; I will update to `undefined`.